tests/integration/common/test_rest_api_service.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+import os 
+
+import pytest
+from common.request import Request
+from aps.APSParams import APSParams
+
+parameters = APSParams().get_params()
+exptected_params = {'base_url': os.getenv('BASE_URL_FOR_APS', 'http://harvest.aps.org/v2/journals/articles'), 'parameters': parameters}
+
+@pytest.fixture
+def rest_api_fixture():
+    return Request(base_url=os.getenv('BASE_URL_FOR_APS', 'http://harvest.aps.org/v2/journals/articles'), parameters=parameters)
+
+@pytest.fixture
+def rest_api_with_params_fixture(rest_api_fixture):
+    rest_api = rest_api_fixture
+    assert exptected_params == rest_api.get()
+    return rest_api
+
+def test_get_response(rest_api_with_params_fixture: Request):
+    reponse = rest_api_with_params_fixture.get_response()
+    assert reponse.status_code == 200
+    
```

Comment:
could you please configure your editor to add a new line at the end of each file?